app.R

```diff
@@ -931,18 +916,120 @@ ui <- fluidPage(
+              "<li> Download Metadata.xlsx with the button below. If adding custom data, add a row and fill in information for each measure you want to add to the Mental Wellness Index. Descriptions for each column can be found in the 'Column Descriptions' sheet of the Metadata.xlsx. Note that <b>all</b> column names, with the exception of 'denominator', must be filled out.</li>",
+              "<ul>",
+              "<li>If you have multiple measures in one file, add a row for each measure and its qualities, but specify the same file name.</li>",
+              "<li>If you would like to remove a measure in your MWI, either delete the measure row or set its weight to 0.</li>",
+              "<li>If you would only like to adjust weights, change only the weight column to the desired values. Note that penalties for race stratifications and geographic granularity are still applied and total weights are scaled to sum to 100.</li>",
+              "</ul>",
+              "<br>",
+              "<li>Upload your Metadata.xlsx and custom data files (if using) and click 'Create Custom MWI' below. Select multiple files by pressing ctrl/cmd and then clicking on each file in the file explorer box. Note: Do not have files open while uploading. This will take some time, depending on the amount of measures included.</li>",
+              "<br>",
+              "<li>Once the custom MWI creation is complete, click 'Download Custom MWI' to download an .RData file with all of the needed information to view your MWI in this tool. <b>Note: if you navigate away from this page, all processing and data will be lost! Nothing is stored within this application.</b></li>",
+              "<br>",
+              "<li>To view your MWI, click the 'Custom MWI Upload' box under 'Explore States' or 'Explore ZIP Codes' and upload the downloaded '.RData' file.</li>",
+              "</ol>",
+              "</p>"
+            )),
+            tagList(
+              hr(),
+              HTML("<center>"),
+              downloadButton("download_metadata", "Download Metadata.xlsx"),
+              HTML("<br><br>"),
+              fileInput(
+                "custom_zip", 
+                "Upload Custom Data Files (.xlsx, .csv) (do not have files open):",
+                accept = c(".xlsx", ".csv"),
+                multiple = T
+              ),
+              actionButton("custom_mwi_go", "Create Custom MWI"),
+              downloadButton("download_custom_mwi", "Download Custom MWI"),
+              HTML("<br><br>"),
+              verbatimTextOutput("custom_error"),
+              HTML("</center>")
+            )
+          ),
+          column(width = 2)
+        )
+      ),
+      tabPanel(
+        title = div("Add Local Data to MWI on Your Computer", class = "explore"),
+        fluidRow(
+          column(width = 2),
+          column(
+            width = 8,
+            HTML("<center><h2>Add local data to create your own Mental Wellness Index (MWI)!</h2></center>"),
```

Comment:
Same here: "Add Local Data to MWI on Your Computer"

Reply:
done!